stream_alert_cli/terraform_generate.py

```diff
@@ -280,6 +281,56 @@ def generate_stream_alert(cluster_name, cluster_dict, config):
 
     return True
 
+def generate_cloudwatch_log_metrics(cluster_name, cluster_dict, config):
+    """Add the CloudWatch Metric Filters module to the Terraform cluster dict.
+
+    Args:
+        cluster_name (str): The name of the currently generating cluster
+        cluster_dict (defaultdict): The dict containing all Terraform config for a given cluster.
+        config (dict): The loaded config from the 'conf/' directory
+
+    Returns:
+        bool: Result of applying the cloudwatch metric filters to the stream_alert module
+    """
+    enable_metrics = config['global'].get('infrastructure',
+                                          {}).get('metrics', {}).get('enabled', False)
+    if not enable_metrics:
+        return True
```

Comment:
Because metrics are optional and returning `False` or `None` would exit the `generate_cluster` function above early which would be bad/no intended. IMHO I don't think the generate functions should return anything if they don't need to. A good amount `return True` with no possibility of ever returning false. See [this comment](https://github.com/airbnb/streamalert/pull/282/files/cdb9197a9ff92e4f2c94ba355882ea5916c62ae7..380819d181c93d456128179932996d9425ac3edb#r136194177)